tests/e2e/e2e_common.go

```diff
@@ -223,6 +223,38 @@ const (
 		"/api-tokens/authorize"
 )
 
+// test suite labels
+const (
+	slow                = "slow"
+	flaky               = "flaky"
+	disruptive          = "disruptive"
+	wcp                 = "wcp"
+	tkg                 = "tkg"
+	vanilla             = "vanilla"
+	topology            = "topology"
+	preferential        = "preferential"
+	vsphereConfigSecret = "vsphereConfigSecret"
+	snapshot            = "snapshot"
+	stable              = "stable"
+	newTests            = "newTests"
+	multivc             = "multivc"
+	block               = "block"
+	file                = "file"
+	core                = "core"
+	P0                  = "P0"
+	P1                  = "P1"
+	P2                  = "P2"
+	vsanStretch         = "vsanStretch"
+	longRunning         = "longRunning"
```

Comment:
what is difference between longrunning and slow labels ?

Reply:
yeah, no difference... removed the slow Label and kept longRunning label